pages/guides.mdx

```diff
@@ -300,7 +300,7 @@ import { AgentsIcon, AgentverseIcon, CliIcon,FetchNetwork, AsiIcon, LedgerIcon }
         path: "/guides/fetch-network/fetch-wallet/mobile-wallet/gov-proposal"
       },
       {
-        title: "Fetch Wallet Hardware Connection Guide",
+        title: "ASI Alliance Wallet Hardware Connection Guide",
         path: "/guides/fetch-network/fetch-wallet/fetch-wallet-hardware-connection-guide"
```

Comment:
Need to update paths to, and update redirects to reflect these changes

Reply:
not clear what you are requiring. Link is working. You mean: /guides/fetch-network/fetch-wallet/fetch-wallet-hardware-connection-guide -> /guides/fetch-network/fetch-wallet/asi-wallet-hardware-connection-guide ?

Reply:
@FelixNicolaeBucsa, I believe we need to update the path, as the title is now ASI Alliance Wallet. The path should be:

`/guides/fetch-network/asi-alliance-wallet/asi-alliance-wallet-hardware-connection-guide`

Reply:
For every link that has been updated can you please update next config to setup redirects? 

like..
```
      {
        source: "/guides/agentverse/registering-agent-services",
        destination:
          "/guides/agentverse/agentverse-functions/registering-agent-services",
        permanent: true,
      },
```

Reply:
Have added all redirects in there for all edited links for wallet, both web and mobile